src/Relation/Unary/Relation/Binary/Equality.agda

```diff
@@ -0,0 +1,34 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Equality of unary relations
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Relation.Unary.Relation.Binary.Equality where
+
+open import Data.Product using (Σ; _,_; swap; zip′)
+open import Function.Base using (id)
+open import Level using (Level)
+open import Relation.Binary.Structures using (IsEquivalence)
+open import Relation.Unary using (_≐_; _≐′_)
+
+private
+  variable
+    a ℓ₁ ℓ₂ : Level
+    A : Set a
+
+≐-isEquivalence : IsEquivalence (_≐_ {A = A} {ℓ₁})
```

Comment:
I think these would be more readable as `IsEquivalence {A = Pred A ℓ₁} _≐_`?

Reply:
Fixed.